sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/AggregateHashMap.java

```diff
@@ -40,7 +42,7 @@
  */
 public class AggregateHashMap {
 
-  private OnHeapColumnVector[] columnVectors;
+  private WritableColumnVector[] columnVectors;
```

Comment:
do we need to change it? I don't think this class is actually used.

Reply:
`columnVectors` is actually referenced several places (e.g. [here](https://github.com/apache/spark/pull/17436/files/9ce6fc0b0ad2c4c97236f0519db07b5a3600bb81#diff-47d4756845a7561110301320621d7f3cR72)) in this class.

Reply:
I mean the `AggregateHashMap` seems only used in the benchmark.

Reply:
Thanks, I realized it this morning. I will revert changes in `AggregateHashMap`.